main.js

```diff
@@ -2,8 +2,13 @@ var receiveMessage = document.querySelector('.receive-button');
 var radioAffirmation = document.querySelector('#affirmation');
 var radioMantra = document.querySelector('#mantra');
 var meditationBell = document.querySelector('#meditate-bell');
-var displayMessage = document.querySelector('#message-display')
-// var message = "";
+var displayMessage = document.querySelector('#message-display');
+var inputName = document.querySelector('#input');
+var welcomePage = document.querySelector('.welcome-box');
```

Comment:
Stretch goal for intermission: querySelect all elements by id attributes

querySelector('#element')
getElementById('element')